core/base/Module.php

```diff
@@ -214,11 +207,8 @@ public function init()
      */
     public function getLayoutPath()
     {
-        if ($this->theme){
-            if ($this->useAppLayoutPath) {
-                $this->setLayoutPath($this->theme->getLayoutPath());
-            }
-            
+        if (Yii::$app->themeManager->hasActiveTheme() && $this->useAppLayoutPath){
+            $this->setLayoutPath(Yii::$app->themeManager->getActiveTheme()->layoutPath);
```

Comment:
for readability i would recommend using `activeTheme->layoutPath` as its a getter method. What do you think?